ClimsoftVer4/ClimsoftVer4/formMetadata.vb

```diff
@@ -1836,6 +1835,53 @@ Err:
         txtClosingDate.Text = ClosingDate.Text
     End Sub
 
+    Private Function DMSToDD(Direction As Char, Deg As String, Min As String, Sec As String) As String
+        ' Convert value in Degrees, Minutes and Seconds (DMS) to Decimal Degrees (DD)
+        ' Direction must be N, S, E or W
+        If IsNumeric(Deg) And IsNumeric(Min) And IsNumeric(Sec) And Not Direction = vbNullChar Then
+            Dim multiplier As Integer = 1
+            Dim decimalDegrees As Double
+            If Direction = "S" OrElse Direction = "W" Then
+                multiplier = -1
+            End If
+            decimalDegrees = multiplier * (Val(Deg) + Val(Min) / 60 + Val(Sec) / 3600)
+            Return Math.Round(decimalDegrees, 2).ToString()
+        Else
+            Return ""
```

Comment:
I've added `return ""` to clear the Lat/Lon whenever any change is begun, otherwise there may be old Lat/Lon values in the form when the user has entered new Deg and Min values (but not entered Sec)

Reply:
@isedwards -Sounds great